aspnetcore/signalr/javascript-client.md

```diff
@@ -98,7 +98,140 @@ Use the [configureLogging](/javascript/api/%40aspnet/signalr/hubconnectionbuilde
 
 ## Reconnect clients
 
-The JavaScript client for SignalR doesn't automatically reconnect. You must write code that will reconnect your client manually. The following code demonstrates a typical reconnection approach:
+::: moniker range=">= aspnetcore-3.0"
+
+### Automatically reconnect
+
+The JavaScript client for SignalR doesn't automatically reconnect by default. Automatic reconnects can be enabled by calling the [withAutomaticReconnect](/javascript/api/%40aspnet/signalr/hubconnectionbuilder#withautomaticreconnect) method on [HubConnectionBuilder](/javascript/api/%40aspnet/signalr/hubconnectionbuilder):
```

Comment:
Feels like you could use language like this here instead since this is a "3.0 forward" thing:

"The JavaScript client for SignalR can be enabled to automatically reconnect using the `withAutomaticReconnect`..."